Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_005.SlackApiKey.sarif

```diff
@@ -54,7 +54,7 @@
           "message": {
             "id": "Default",
             "arguments": [
-              "xoxb-8…",
```

Comment:
>xoxb-8 [](http://example.com/codeflow?start=15&length=6)

Here's a great example of why we do this, observe there is only a single character of randomization in this token truncation.